<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DoctorWindow</class>
 <widget class="QMainWindow" name="DoctorWindow">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>797</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MCA Doctor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="patientsTable">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>519</width>
      <height>399</height>
     </rect>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="dragDropOverwriteMode">
     <bool>false</bool>
    </property>
    <property name="alternatingRowColors">
     <bool>true</bool>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QLabel" name="titleLbl">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>281</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#4b0082;&quot;&gt;Hello &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LogOut">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>520</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Log Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="addPatientBtn">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>150</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Add Patient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewPatientBtn">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>194</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>View Patient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="editPatientBtn">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>106</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Edit Patient</string>
    </property>
   </widget>
   <widget class="QPushButton" name="deletePatientbtn">
    <property name="geometry">
     <rect>
      <x>621</x>
      <y>238</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Delete Patient</string>
    </property>
   </widget>
   <widget class="QCommandLinkButton" name="commandEditPersonalInfo">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>185</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Edit Pesonal Info</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>797</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionEdit_Personal_Info"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionEdit_Personal_Info">
   <property name="text">
    <string>Edit Personal Info</string>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>actionEdit_Personal_Info</sender>
   <signal>changed()</signal>
   <receiver>DoctorWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>398</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
